winui-toc.html

```diff
@@ -85,6 +85,16 @@
 			<li><a href="/winui/AutoComplete/Highlighting">Highlighting</a></li>
         </ul>
     </li>
+    <li>
+        AvatarView
+       <ul>
```

Comment:
@MadhumithanJayapathi Use small letters for href link.

Reply:
@Manivannan-E  Used small letters for href links